### PR TITLE
Adam dashboard

### PR DESCRIPTION
**Pull request by AdamBergstrom138**

### CODE REVIEW

File: src/components/Dashboard/DashboardTable1.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+return (
+    <tr>
+        <td>{client.company_name}</td>  
+        <td>{client.status}</td>
+        <td><button type="button" class="btn btn-primary" onClick={() => handleClick(client)}>Edit</button></td>
+    </tr>
+)
+}
+
+export default DashboardTable1;
```
Comment (laheavey) [inline src/components/Dashboard/DashboardTable1.jsx:23]: If I'm tracking, DashboardTable1 is the 'Edit In Progress' status clients, and DashboardTable2 is the 'Active' status clients by Operator ID, right? 

Maybe change the names to something more descriptive? I also didn't catch any logic that would route the 'Edit in Progress' ones to DT1 and 'Active' ones to DT2, but I might have missed it/might be thinking too far ahead!

File: server/routes/client.router.js [hunk 2]
```diff
@@ -33,7 +67,31 @@ router.get('/all', (req, res) => {
 });
 
 router.post('/', (req, res) => {
-  // POST route code here
+  console.log(req);
```
Comment (laheavey) [inline server/routes/client.router.js:70]: are the GET routes the only ones we put 'rejectUnauthenticated' on? Or would we put them on the POST/PUT/DELETEs too? Genuinely asking, not sure but I think maybe yes on all - only flagging as an FYI.

File: server/routes/client.router.js [hunk 3]
```diff
@@ -33,7 +67,31 @@ router.get('/all', (req, res) => {
...
+  const queryValues = [
+    newCompany.companyName,
+    newCompany.contactPerson,
+    newCompany.emailInput
+  ];
+  pool.query(queryText, queryValues)
+    .then((result) => { 
+      console.log('client post id:', result.rows[0].id);
+      res.sendStatus(201);
+      res.send(result.rows[0]);
```
Comment (laheavey) [inline server/routes/client.router.js:89]: Do you have to send the result back, or do you just need the sendStatus(201)?

File: src/redux/sagas/client.saga.js [hunk 4]
```diff
@@ -36,8 +37,33 @@ function* deleteClient(action) {
   }
 };
 
+function* postClient(action) {
+  try {
+    const newCompany = action.payload
+    console.log(action.payload);
+    const clientresoponce = yield axios({
```
[the remark this answers is not in the document]
Comment (AdamBergstrom138) [reply src/redux/sagas/client.saga.js:44]: resolving now!  Good catch it was late last night when I was working on this lol.  Thank you!

File: src/redux/sagas/client.saga.js [hunk 5]
```diff
@@ -36,8 +37,33 @@ function* deleteClient(action) {
...
+      method: 'POST',
+      url: '/api/client',
+      data: newCompany
+    })
+      console.log('new clientyeid', clientresoponce);
```
[the remark this answers is not in the document]
Comment (AdamBergstrom138) [reply src/redux/sagas/client.saga.js:49]: Resolving now!

File: src/redux/sagas/client.saga.js [hunk 6]
```diff
@@ -36,8 +37,33 @@ function* deleteClient(action) {
...
+      yield axios({
+        method: 'POST',
+        url: '/client-assessment',
+        data: {...newCompany, client_id: clientresoponce.id}
```
[the remark this answers is not in the document]
Comment (AdamBergstrom138) [reply src/redux/sagas/client.saga.js:53]: Resolving now!

File: src/components/Dashboard/DashboardTable1.jsx (hunk 1 above)
Comment (AdamBergstrom138) [reply src/components/Dashboard/DashboardTable1.jsx:23]: I'm adding the logic for the buttons in now.  After my tables blew up I've been focusing on that but you are correct.  I can change the names to be more expressive as well!

File: server/routes/client.router.js (hunk 3 above)
Comment (AdamBergstrom138) [reply server/routes/client.router.js:89]: I'm not sure If I do but I don't think it'll hurt anything and while I'm still testing I'm going to leave it in until it's fully finished!  Thanks so much for your feedback!

**Pull request closed without merging**